bundles/remote_services/remote_service_admin_dfi/src/remote_service_admin_dfi.c

```diff
@@ -555,8 +633,16 @@ celix_status_t remoteServiceAdmin_removeExportedService(remote_service_admin_t *
             arrayList_destroy(exports);
         }
 
-        exportRegistration_close(registration);
-        exportRegistration_destroy(registration);
+        if (CELIX_RSA_USE_STOP_EXPORT_THREAD) {
```

Comment:
This is a code clone from above. Perhaps refactor into a macro or a function?

Reply:
done